content/docs/rfcs/16/README.md

```diff
@@ -117,23 +117,6 @@ none
 
 Refer to the [Waku Relay specification](https://github.com/vacp2p/specs/blob/master/specs/waku/v2/waku-relay.md) for more information on the relaying of messages.
 
-### Types
-
-The following structured types are defined for use on the Relay API:
-
-#### WakuRelayMessage
-
-`WakuRelayMessage` is an `Object` containing the following fields:
-
-| Field | Type | Inclusion | Description |
-| ----: | :---: | :---: | ----------- |
-| `payload` | `String` | mandatory | The payload being relayed as a hex encoded data string |
-| `contentTopic` | `String` | optional | Message content topic for optional content-based filtering |
-| `timestamp` | `Number` | optional | The time at which the message is generated by its sender. This field holds the Unix epoch time in nanoseconds as a 64-bits integer value.  |
-| `ephemeral` | `Boolean` | optional | This flag indicates the transient nature of the message. |
```

Comment:
This API input `Object` was constructed to separate the fields that a user/app may want to provide to the API from what the API should provide itself. This required separating the `WakuMessage` wire format from the object provided to `post` calls. Currently the only field which is determined by the API is `version`, but wouldn't it make sense to maintain separation between what is published and the message constructed by the node/API?